scripts/dockerbuild

```diff
@@ -5,6 +5,9 @@
 # By default, the host user ID and group ID are forwarded through
 # build arguments. To disable this behavior and use the dockerfile
 # defaults, set the SWG_DEFAULT_USER environment variable to 1.
+#
+# The hugo image can be built with the SWG_HUGO_ENV_ARG environment variable
```

Comment:
I would suggest a more targeted name like `SWG_VERSION_INFO` (or something similar) And the corresponding modification of `HUGO_ENV_ARG`.

Reply:
I've now updated the names/used different names as we have different variables. I'm not attached to them though so feel free to commit alternatives if you prefer.